lib/playwright/channel_owners/browser_context.rb

```diff
@@ -216,15 +208,6 @@ def set_offline(offline)
     end
 
     def add_init_script(path: nil, script: nil)
-      source =
-        if path
-          File.read(path)
-        elsif script
-          script
-        else
-          raise ArgumentError.new('Either path or script parameter must be specified')
-        end
-
```

Comment:
This is also my fault...
source must be set to addInitScript.
https://github.com/YusukeIwaki/playwright-ruby-client/pull/210